app/templates/dashboard.hbs

```diff
@@ -3,8 +3,16 @@
 <div id='crates-heading'>
     {{svg-jar "dashboard"}}
     <h1>My Dashboard</h1>
+    <div id="stats">
+        <div class='downloads'>
+            <img class="download" src="/assets/download.svg" />
```

Comment:
FWIW this should probably be using `{{svg-jar "download"}}` instead of the explicit `<img>`, which will include the SVG as an inline element